pkg/types/conversion/installconfig.go

```diff
@@ -79,11 +82,19 @@ func ConvertNetworking(config *types.InstallConfig) {
 	}
 }
 
-// ConvertBaremetal upconverts deprecated fields in the baremetal
+// convertBaremetal upconverts deprecated fields in the baremetal
 // platform. ProvisioningDHCPExternal has been replaced by setting
 // the ProvisioningNetwork field to "Unmanaged"
-func ConvertBaremetal(config *types.InstallConfig) {
+func convertBaremetal(config *types.InstallConfig) {
 	if config.Platform.BareMetal.DeprecatedProvisioningDHCPExternal == true && config.Platform.BareMetal.ProvisioningNetwork == "" {
 		config.Platform.BareMetal.ProvisioningNetwork = baremetal.UnmanagedProvisioningNetwork
 	}
 }
+
+// convertOpenStack upconverts deprecated fields in the OpenStack platform.
+func convertOpenStack(config *types.InstallConfig) {
+	// LbFloatingIP has been renamed to APIFloatingIP
+	if config.Platform.OpenStack.DeprecatedLbFloatingIP != "" && config.Platform.OpenStack.APIFloatingIP == "" {
+		config.Platform.OpenStack.APIFloatingIP = config.Platform.OpenStack.DeprecatedLbFloatingIP
```

Comment:
I'd expect to clear the old property when you shuffle the value over.

Reply:
I think we should keep it to show a warning message later, during install config validation.

Reply:
> I'd expect to clear the old property when you shuffle the value over.

Don't reset user set values, if a user set the deprecated value, we can use that to set the new values but do not modify the value set by the user.